yesod-form/Yesod/Form/Functions.hs

```diff
@@ -615,3 +618,23 @@ convertField to from (Field fParse fView fEnctype) = let
   fParse' ts = fmap (fmap (fmap to)) . fParse ts
   fView' ti tn at ei = fView ti tn at (fmap from ei)
   in Field fParse' fView' fEnctype
+
+-- | Removes a CSS class from the 'fsAttrs' in a 'FieldSettings'.
+--
```

Comment:
I think this would be a great place for a concrete usage example, it's unclear from the docs or signature exactly what the behavior of this is supposed to be.